public/app/features/migrate-to-cloud/MigrateToCloud.tsx

```diff
@@ -33,6 +33,15 @@ export default function MigrateToCloud() {
           </TextLink>{' '}
           to learn more about this feature!
         </Trans>
+        {config.cloudMigrationIsTarget && (
+          <>
+            &nbsp;
+            <Trans i18nKey="migrate-to-cloud.public-preview.message-cloud">
+              Your self-managed instance of Grafana requires version 11.5+, or 11.2+ with the onPremToCloudMigrations
```

Comment:
nit: if we have a monospace font with background (similar to GH's `inline code`), we might want to use this for the flag